charts/trino/templates/ingress.yaml

```diff
@@ -0,0 +1,61 @@
+{{- if .Values.ingress.enabled -}}
+{{- $fullName := include "trino.fullname" . -}}
+{{- $svcPort := .Values.service.port -}}
+{{- if and .Values.ingress.className (not (semverCompare ">=1.18-0" .Capabilities.KubeVersion.GitVersion)) }}
```

Comment:
@hashhar should we consider only supporting greater than not supported by Kubernetes, as per [the Trino documentation](https://trino.io/docs/current/installation/kubernetes.html#requirements).

Reply:
Therefore, we should be using the latest api.

```
apiVersion: networking.k8s.io/v1
```